content/docs/command-reference/checkout.md

```diff
@@ -18,9 +18,9 @@ positional arguments:
 ## Description
 
 This command is usually needed after `git checkout`, `git clone`, or any other
-operation that changes the current `dvc.lock` or `.dvc` files. It restores the
-corresponding versions of the DVC-tracked files and directories from the
-<abbr>cache</abbr> to the workspace.
+operation that changes the current [DVC files](/doc/user-guide/dvc-files). It
+restores the corresponding versions of the DVC-tracked files and directories
+from the <abbr>cache</abbr> to the workspace.
```

Comment:
This is the only generalization done in this PR. Reviewing all the instances of these repetitive mentions will require a separate PR and is related to https://github.com/iterative/dvc.org/issues/1663#issuecomment-749333023